spec/dummy/config/initializers/rad_common.rb

```diff
@@ -49,3 +49,5 @@
       query_where: 'divisions.name ilike :search',
       super_search_exclude: true }
   ]
+
+ActionController::Base.send :helper, RadCommon::BootstrapFlashHelper
```

Comment:
This looks weird, was there a problem with this helper being accessible?

Reply:
Yea. The helper isn't included by default. We could have the module extend itself and then call the helper function on the module itself in the layout. But that also is not commonly seen in a rails developer's perspective.

Reply:
I removed this and it seems to work, please confirm.